### PR TITLE
Matching exact score doesn't end game

### LINKED ISSUE #533: Matching exact score doesn't end game
**Describe the bug**
When the player has exactly the target score, the game does not end.

**To Reproduce**
Steps to reproduce the behaviour:
1. Play game
2. Get the exact target score.
3. Game does not end.

**Expected behavior**
The game ends.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by maimoona-bashir**
Updated WinnerModal.vue

Fixes #533

#### Overview of changes

- game ends when the score is less than target score

- 

-

Reviewer: @johnanvik

### CODE REVIEW

File: src/components/Modals/WinnerModal.vue [hunk 1]
```diff
@@ -138,7 +138,7 @@ export default {
       let winList = []
       for (let player of this.playerList) {
         if (this.getActiveSide()) {
-          if (this.getScore(this.getCurrentPlayer().id).score >= this.getScoreLimit()) {
```
Comment (johnanvik) [inline src/components/Modals/WinnerModal.vue:141]: This looks like it is correct.

File: src/components/Modals/WinnerModal.vue [hunk 2]
```diff
@@ -138,7 +138,7 @@ export default {
...
+          if (this.getScore(this.getCurrentPlayer().id).score > this.getScoreLimit()) {
```
Comment (johnanvik) [inline src/components/Modals/WinnerModal.vue:141]: This looks it would cause the bug.

**Comment by johnanvik** [comment]
Suggestion for testing:
 * Change the player's hand to have the same card(s) at the start every game (e.g. Instruction 2)
 * Change the player's starting score to almost 75 (e.g. 73)
 * Play game
 * Play instruction card (e.g. I2)
 * Confirm that game does not end as it should

**Comment by maimoona-bashir** [comment]
Scores

**Comment by maimoona-bashir** [comment]
scores2
  

working

**Comment by johnanvik** [comment]
This is actually working. The 105 score is because of the bonuses. The important thing is that the instruction score is 75 and the game ends - which it does.

**Comment by johnanvik** [comment]
Closing as this appears to have not have been a bug.

**Pull request merged**